src/main/java/com/opencart/steps/RegisterPageBL.java

```diff
@@ -72,23 +72,51 @@ public RegisterPageBL registerNewPersonWithoutEmail() {
         inputFirstName(registerModel.getFirstName());
         inputLastName(registerModel.getLastName());
         inputTelephone(registerModel.getTelephone());
-        inputPassword(registerModel.getPassword());
+        inputPasswordAndConfirmPassword(registerModel.getPassword());
         clickPolicyCheckbox();
         clickOnContinueButton();
         return this;
     }
 
-    public RegisterPageBL regiterNewPersonWithoutTelephone() {
+    public RegisterPageBL registerNewPersonWithoutTelephone() {
         RegisterModel registerModel = RegisterModelRepository.getRegisterModel();
         inputFirstName(registerModel.getFirstName());
         inputLastName(registerModel.getLastName());
         inputEmail(registerModel.getEmail());
+        inputPasswordAndConfirmPassword(registerModel.getPassword());
+        clickPolicyCheckbox();
+        clickOnContinueButton();
+        return this;
+    }
+
+    public RegisterPageBL registerNewPersonWithoutPassword() {
+        RegisterModel registerModel = RegisterModelRepository.getRegisterModel();
+        inputFirstName(registerModel.getFirstName());
+        inputLastName(registerModel.getLastName());
+        inputEmail(registerModel.getEmail());
+        inputTelephone(registerModel.getTelephone());
+        clickPolicyCheckbox();
+        clickOnContinueButton();
+        return this;
+    }
+
+    public RegisterPageBL registerNewPersonWithoutVerifyPassword() {
+        RegisterModel registerModel = RegisterModelRepository.getRegisterModel();
+        inputFirstName(registerModel.getFirstName());
+        inputLastName(registerModel.getLastName());
+        inputEmail(registerModel.getEmail());
+        inputTelephone(registerModel.getTelephone());
```

Comment:
I believe you can set RegisterModel as a parameter into registerNewPerson() method. So, as a result, you won't need to create a few registerNewPerson() methods for this, you will just put corresponding models from test